network/ipam/how-to/reserve-ip.mdx

```diff
@@ -46,8 +46,32 @@ This page sets out the steps necessary to reserve an IP address with IPAM.
 
 4. Define whether you want to specify the IP address to reserve, or reserve any available IP on the Private Network (the IP selected will be displayed after reservation). If specifying the IP address yourself, do **not** include the network prefix (e.g. `/22`).
 
+5. Define whether you want to attach a MAC address to the reserved IP. **Only do this for custom resources** e.g. virtual machines hosted on a Proxmox cluster on an Elastic Metal server. Do **not** attach a MAC address for reserved IPs you want to use with standard Scaleway resources such as Instances, Load Balancers or Elastic Metal servers themselves.
```

Comment:
Maybe the "do not" part can be inside an important message box to make it stand out more?